docs/src/integrations/exchange.md

```diff
@@ -273,7 +273,10 @@ can request the block from RPC in binary format, and parse it using either our
 
 ### Address History
 
-You can also query the transaction history of a specific address.
+You can also query the transaction history of a specific address. This is
+generally *not* a viable method for tracking all your deposit addresses over all
+slots, but may be useful for examining a few accounts for a specific period of
+time.
 
 - Send a [`getConfirmedSignaturesForAddress`](developing/clients/jsonrpc-api.md#getconfirmedsignaturesforaddress)
```

Comment:
nits: this is deprecated? So, let's reference `getConfirmedSignaturesForAddress2`?